esp_encrypted_img/CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+## 2.0.5
+
+### Enhancements:
+- Added an API to abort the decryption process: `esp_encrypted_img_decrypt_abort`
+
+### Breaking Changes:
+- `esp_encrypted_img_decrypt_end()` API now also returns `ESP_ERR_INVALID_STATE` error. In such case, memory used by the context is not freed by `esp_encrypted_img_decrypt_end()`. Use `esp_encrypted_img_decrypt_abort()` API to free up the memory used by context in case of any errors.
```

Comment:
We should also talk about using `esp_encrypted_img_decrypt_data` in this scenario as a recovery method. Could we call this as a bugfix, where an early call to `esp_encrypted_img_decrypt_end` would not free up entire context and allows a recovery mechanism?

Reply:
Yes. I have updated the PR with the changes.